README.md

```diff
@@ -53,8 +53,20 @@ The tools available in FRE-NCtools are:
 * runoff_regrid -- Regrid land runoff data to the nearest ocean point
 * transfer_to_mosaic_grid -- Convert older style grids to newer mosaic grid
 
+### Other tools
+The [ Ocean Model Grid Generator](https://github.com/NOAA-GFDL/ocean_model_grid_generator)
+is a collection of tools for creating finite element spherical tripolar grids for
+GFDL's MOM based ocean models. Unlike the other tools, NCTools includes it as a submodule,
+and also it is a Python3 project. Because of the former attribute, recursive
+cloning (see below) is recommended. Because of the latter attribute,
+the users python environment may need modification and/or options to the
+autotools configure command may need to be specified. This includes the
+--[enable|disable]-ocean-model-grid-generator option (default is enable)
```

Comment:
Please place these options within backticks.  This will help identify them as options.  For example:

<pre>
This includes the `--disable-ocean-model-grid-generator` option ... .
</pre>

Also, when an option is needed that enables or disables, or with or without, then include the option that is not the default.  In this case, the option is `--disable-ocean-model-grid-generator`.

Reply:
@underwoo 
OK.  Leat me know if the latest push needs further improvements. Thanks.